resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -387,6 +400,11 @@ Sometimes a kernel loads, but doesn't work as expected. To test whether your ins
 
 ![](/resources_pages/imgs/jupyter_lab_r_kernel2.png)
 
+> **Note:** It might take a long time to open JupyterLab
+> when there is an R kernel notebook open
+> since the previous session.
+> Be patient and wait several seconds for JupyterLab to start the R kernel.
```

Comment:
I am not sure if this will be an annoyance during the semester and if it is something we can solve easily.